creator-node/src/snapbackSM/computeSyncModeForUserAndReplica.js

```diff
@@ -0,0 +1,116 @@
+const DBManager = require('../dbManager.js')
+const { asyncRetry } = require('../utils.js')
+
+/**
+ * Sync mode for a (primary, secondary) pair for a user
+ */
+const SyncMode = Object.freeze({
+  None: 'NONE',
+  SecondaryShouldSync: 'SECONDARY_SHOULD_SYNC',
+  PrimaryShouldSync: 'PRIMARY_SHOULD_SYNC'
+})
+
+const FetchFilesHashNumRetries = 3
```

Comment:
maybe we want to make this a config value

Reply:
don't think its worth since this is not something we'll really need to change at the node level, it can be any arbitrary value, i'm hesitant to pollute config space with unnecessary vals